crates/matrix-sdk-ui/src/timeline/builder.rs

```diff
@@ -172,7 +172,7 @@ impl TimelineBuilder {
         let (room_event_cache, event_cache_drop) = room.event_cache().await?;
         let (_, event_subscriber) = room_event_cache.subscribe().await;
 
-        let is_live = matches!(focus, TimelineFocus::Live { .. });
+        let is_live = matches!(focus, TimelineFocus::Live { .. } | TimelineFocus::Thread { .. });
```

Comment:
Can we view thread on a pinned event? Can we view thread on a permalink? In those case, `is_live` would be `true` and I think it's an error, thought?